test/controllers/secrets_controller_test.rb

```diff
@@ -507,5 +516,39 @@ def do_update(extras = {})
         Samson::Secrets::DbBackend::Secret.exists?(secret.id).must_equal(false)
       end
     end
+
+    describe "#resolve" do
+      it "renders resolved secrets" do
+        create_secret 'production/z/pod2/bar'
+        get :resolve, params: {project_id: other_project.id, deploy_group: 'pod2', keys: ['bar', 'foo'], format: 'json'}
+        assert_response :success
+
+        result = JSON.parse(response.body)
```

Comment:
FYI usually I go with `must_equal` for these cases so it's easier to see what the expected output is and extra keys get shown too

Reply:
i'm not exactly following -- can you point to an example?